src/utils/bridge.js

```diff
@@ -5,6 +5,7 @@ import parseException from './exception-parser';
 import { debug } from './logger';
 import { isDevMode } from './dev-mode';
 import { basicFetch } from './fetch';
+import { getBlockTypes } from '@wordpress/blocks';
```

Comment:
While rebasing https://github.com/wordpress-mobile/GutenbergKit/pull/197, I discovered this import is problematic for the "remote" editor, which currently fails to load. The import results in a reference to `window.wp.blocks` before this global is defined by scripts loaded from the remote site. 

I'll create a PR addressing this, likely by splitting (or "chunking") this bridge method from the rest.